src/db/sysdb_search.c

```diff
@@ -2430,18 +2430,19 @@ int sysdb_get_netgroup_attr(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
-errno_t sysdb_get_direct_parents(TALLOC_CTX *mem_ctx,
-                                 struct sss_domain_info *dom,
-                                 struct sss_domain_info *parent_dom,
-                                 enum sysdb_member_type mtype,
-                                 const char *name,
-                                 char ***_direct_parents)
+errno_t sysdb_get_direct_parents_ex(TALLOC_CTX *mem_ctx,
+                                    struct sss_domain_info *dom,
+                                    struct sss_domain_info *parent_dom,
+                                    enum sysdb_member_type mtype,
+                                    const char *name,
+                                    const char *attr_name,
+                                    char ***_direct_parents)
 {
     errno_t ret;
     const char *dn;
     char *sanitized_dn;
     struct ldb_dn *basedn;
-    static const char *group_attrs[] = { SYSDB_NAME, NULL };
+    static const char *group_attrs[] = { SYSDB_NAME, NULL, NULL };
```

Comment:
Why do we need both `SYSDB_NAME` and `attr_name`?

Maybe simply:
```
if (!attr_name) attr_name = SYSDB_NAME;
group_attrs[0] = group_attrs;
```
and that's it?

Btw, no real point to have this 'static'.

Reply:
Thanks for the suggestion, fixed.